index.js

```diff
@@ -35,7 +37,7 @@ function deploy (options) {
     if (config.forceDeploy) {
       return runDeploy(config);
     }
-    return openshift(config, {request: { strictSSL: options.strictSSL }}).then(client => {
+    return openshift({config: config, request: { strictSSL: options.strictSSL }}).then(client => {
```

Comment:
Nit: Why not keep the property shorthand?

Reply:
it's actually changing the signature to be 1 param instead of 2.

but, yea, i can still make that use the shorthand notation.  i'll update that